test/MochaSpecRunner.html

```diff
@@ -0,0 +1,51 @@
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>Mocha Tests</title>
+    <script type="text/javascript" src="lib/webcomponents.js"></script>
+    <link href="lib/mocha.css" rel="stylesheet" />
+    <link rel="import" href="../lab/polymer/components/my-menu-button.html">
+</head>
+
+<body>
+    <my-menu-button style="position: absolute; top: 0; right: 0" href="subpage.html">Distributed <strong>Content</strong></my-menu-button>
+    <div id="brand">
+        <img src="tests-logo.png" style="max-width: 100%; margin: 20px 0px 0px 20px" />
+    </div>
+    <div id="mocha"></div>
+
+    <script src="lib/jquery.min.js"></script>
```

Comment:
Do you know where exactly? I have removed the above line and all tests still pass, the HTML reporter works.